sys-apps/config-site/files/config.site.in

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+for script in "@datarootdir@/config.site.d"/*; do
```

Comment:
Let's not repeat the Portage mistake again and make this something like:

```suggestion
for script in "@datarootdir@/config.site.d"/*.conf; do
```

(or `.site`)

Reply:
I changed it to `*.conf`.